vcluster/manage/upgrade.mdx

```diff
@@ -0,0 +1,136 @@
+---
+title: Upgrade vCluster
+sidebar_label: Upgrade
+sidebar_position: 4
+---
+
+import Tabs from '@theme/Tabs'
+import TabItem from '@theme/TabItem'
+
+<Tabs
+  groupId="get-started"
+  defaultValue="cli"
+  values={[
+	{ label: "vCluster CLI", value: "cli" },
+    { label: 'Helm', value: 'helm', },
+    { label: 'Terraform', value: 'terraform', },
+    { label: 'Argo CD', value: 'argo', },
+    { label: 'Cluster API', value: 'cluster-api', },
+  ]
+}>
+<TabItem value="cli">
+
+Upgrade your CLI to the version of vCluster that you want to upgrade your virtual cluster to.
+
+```bash
+vcluster upgrade --version <VCLUSTER_VERSION>
+vcluster create --upgrade VCLUSTER_NAME -n VCLUSTER_NAMESPACE -f vcluster.yaml
+```
+
+Replace:
+
+- _`VCLUSTER_VERSION`_ with the vCluster version that you want to upgrade to
+- _`VCLUSTER_NAME`_ with your vCluster instance name.
+- _`VCLUSTER_NAMESPACE`_ with the namespace where you deployed vCluster.
+
+</TabItem>
+<TabItem value="helm">
+
+Since the v0.20 release is a beta, you need to specify the version or Helm uses the latest GA version.
+
+```bash
+helm upgrade --install VCLUSTER_NAME vcluster \
+  --values vcluster.yaml \
+  --repo https://charts.loft.sh \
+  --namespace VCLUSTER_NAMESPACE \
+  --repository-config='' \
+  --version 0.20.0-beta.12
+```
+
+Replace:
+
+- _`VCLUSTER_NAME`_ with your vCluster instance name.
+- _`VCLUSTER_NAMESPACE`_ with the namespace where you deployed vCluster.
+
+</TabItem>
+<TabItem value="terraform">
+
+Apply vCluster config changes by editing the `vcluster.yaml` file and running `terraform plan`:
+
+```bash
+terraform plan
+```
+
+Review the planned changes and apply them if they look appropriate:
+
+```bash
+terraform apply
+```
+
+</TabItem>
+<TabItem value="argo">
+
+Add your `vcluster.yaml` config file to the `valueFiles` array in your ArgoCD `Application` file.
+
+ Since the v0.20 release is a beta, you need to specify the `targetRevision` or Helm uses the latest GA version.
```

Comment:
nit: Could add this "Since the v0.20..." verbiage to an [admonition](https://github.com/zerbitx/vcluster-docs/tree/main/vcluster/_partials/admonitions) partial for reuse.